src/Microsoft.AspNet.Localization/RequestLocalizationOptions.cs

```diff
@@ -25,20 +29,40 @@ public RequestLocalizationOptions()
         }
 
         /// <summary>
-        /// The cultures supported by the application. If this value is non-<c>null</c>, the
-        /// <see cref="RequestLocalizationMiddleware"/> will only set the current request culture to an entry in this
-        /// list. A value of <c>null</c> means all cultures are supported.
-        /// Defaults to <c>null</c>.
+        /// Gets or sets the default culture to use for requests when a supported culture could not be determined by
+        /// one of the configured <see cref="IRequestCultureProvider"/>s.
+        /// Defaults to <see cref="CultureInfo.CurrentCulture"/> and <see cref="CultureInfo.CurrentUICulture"/>.
+        /// </summary>
+        public RequestCulture DefaultRequestCulture
+        {
+            get
+            {
+                return _defaultRequestCulture;
+            }
+            set
+            {
+                if (value == null)
+                {
+                    throw new ArgumentNullException();
```

Comment:
Yeah I actually wasn't sure what name to use here, but property name makes sense.